Removed legacy Cloud infrastructure info (#5125)
magento/devdocs: guides/v2.2/cloud/project/project-webint-branch.md

This topic discusses how to use the Project Web Interface to:

- Add or delete an environment. You cannot add or delete the `master` branch for Pro plan Staging and Production. You can branch from Start plan Master.
- Sync (`git pull`) from the environment's parent
- Merge (`git push`) to the environment's parent

{:.bs-callout .bs-callout-info}
You cannot create branches from Pro plan Staging and Production `master`. These environments include `master` branches that you deploy updated Git code to from Integration `master`.

## Add or delete an environment {#project-branch-add}

Complete development of code and added extensions in a branch and, when complete, merge (`git push`) the branch with its parent or master.

- For Starter, we recommend you create a `staging` branch from Master, then branch from `staging` for development.

> - For Pro, you want to branch from the development branch (Integration `master`).

Reviewer: ```suggestion
- For Pro, create a development branch from the Integration environment.
```
